frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -83,9 +83,10 @@ export function parsePeopleParams(peopleParams: PeopleParamType, filters: Partia
     const { action, date_from, date_to, breakdown_value, ...restParams } = peopleParams
     const params = filterClientSideParams({
         ...filters,
-        entity_id: action.id || filters?.events?.[0]?.id || filters?.actions?.[0]?.id,
-        entity_type: action.type || filters?.events?.[0]?.type || filters?.actions?.[0]?.type,
-        entity_math: action.math || undefined,
+        entity_id: (typeof action !== 'string' && action.id) || filters?.events?.[0]?.id || filters?.actions?.[0]?.id,
```

Comment:
is there a better way about this? :| somewhere we're passing in an object or a string to this action value and I don't know if there's a cleaner way to fix the typing for it, without having to dig deep and refactor where this comes from

Reply:
What if the action is assigned to a temporary variable iff it's a string, and what is now called `action` is guaranteed to be an object? `id` could be null if the original input was a string, because id should be non-nullable, right?

Reply:
This looks fine. Just not sure why action is a string in the first place :).